src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/tests/Action.test.js

```diff
@@ -19,24 +19,6 @@ test('The component should call the callbacks after a click', () => {
     expect(afterAction).toBeCalled();
 });
 
-test('The component should call the callbacks after press enter', () => {
-    const onClick = jest.fn();
-    const afterAction = jest.fn();
-    const action = shallow(<Action afterAction={afterAction} onClick={onClick} value="my-option">My action</Action>);
-    action.find('button').simulate('keydown', {key: 'Enter', preventDefault: jest.fn(), stopPropagation: jest.fn()});
-    expect(onClick).toBeCalled();
-    expect(afterAction).toBeCalled();
-});
-
-test('The component should call the callbacks after press space', () => {
```

Comment:
Same for this can we adjust them so we test that the navigation works like expected?